packages/forms-web-app/src/dynamic-forms/journey.js

```diff
@@ -68,6 +71,11 @@ class Journey {
 		}
 		this.listingPageViewPath = listingPageViewPath;
 
+		if (!journeyTitle && typeof journeyTitle !== 'string') {
```

Comment:
i've copied logic from other fields above this one, but not sure it is correct

Reply:
yep, sorry if that was my silly mistake